charts/dawarich/values.yaml

```diff
@@ -104,6 +105,52 @@ photon:
 #    limits:
 #      memory: 512Mi
 
+  loadbalancerProxy:
+    image:
+      registry: docker.io
+      repository: envoyproxy/envoy
+      tag: v1.31-latest
+      pullPolicy: IfNotPresent
+    podSecurityContext: {}
+    photonHosts:
+      - host: photon.komoot.io
+        scheme: https
+        port: 443
+        rateLimit: 0  # no known limit
+        preferred: false
+      - host: photon.koalasec.org
+        scheme: https
+        port: 443
+        rateLimit: 0  # no known limit
+        preferred: false
+      - host: photon.donsomhong.net
+        scheme: https  
+        port: 443
+        rateLimit: 10  # 10 requests per second per IP
+        preferred: false
+      - host: photon.marsmathis.com
+        scheme: https
+        port: 443
+        rateLimit: 0  # no known limit
+        preferred: false
+      - host: photon.vanoosterhout.cloud
+        scheme: https
+        port: 443
+        rateLimit: 10  # 10 requests per second per IP
+        preferred: false
+      - host: photon.kllswitch.com
+        scheme: https
+        port: 443
+        rateLimit: 0  # no known limit
+        preferred: false
+
+#    resources:
```

Comment:
Are these resource requests and limits realistic for envoy? By adding them as a comment these are recommendations. The request/response should be very low since it's simple json without transformations so this could be realistic but if not, don't specify.